src/style.css

```diff
@@ -1 +1,31 @@
-/* styles go here */
+body {
+  margin: 0 8px;
+}
+
+div {
+  margin: 20px 0;
+}
+
+fieldset {
+  padding-top: 0;
+  padding-bottom: 0;
+}
```

Comment:
Checklist, point 6

Reply:
And by the way, I guess it's unnecessary to reset fieldset paddings. Just add this styles

![Screenshot 2023-01-23 at 18 08 56](https://user-images.githubusercontent.com/84275496/214089369-5cf9d8e5-054f-4526-a363-72f8116e5d3f.png)